crytic_compile/platform/etherscan.py

```diff
@@ -298,6 +298,10 @@ def compile(self, crytic_compile: "CryticCompile", **kwargs: str) -> None:
             if "message" not in info:
                 LOGGER.error("Incorrect etherscan request")
                 raise InvalidCompilation("Incorrect etherscan request " + etherscan_url)
+            
+            if not info["message"].startswith("OK") and "Invalid API Key" in info["result"]:
```

Comment:
hi! this is a fix for the CI linting failure

```suggestion

            if not info["message"].startswith("OK") and "Invalid API Key" in info["result"]:
```

Reply:
i missed it, thank mate @elopez